graphdatascience/endpoints.py

```diff
@@ -61,7 +61,7 @@ def __getattr__(self, attr: str) -> IndirectAlphaCallBuilder:
 
 """
 This class should inherit endpoint classes that only expose calls of the `gds.beta` namespace.
```

Comment:
this replace does not make sense as its for BetaEndpoints. (same above for alpha)